### DISCUSSION TITLE
Arcticons 10

### DISCUSSION THREAD

**Discussion by Donnnno** [Arcticons-Team/Arcticons#2204] [Announcements]
###### Released: Jul 21, 2024
* 🌈 Another special release, we've reached the **10k**!
* 🎉 **332** new and updated icons!
* 🎮 A new **Game** category.
* 📖 Our [**Knowledge base**](https://docs.arcticons.com/) has been upgraded.
* 🔥 **10059** icons in total!

![Artboard 333](https://github.com/user-attachments/assets/430ddcfe-290b-4114-a928-6e4554f8afa1)

-------------------

## What's Changed
* add lichess beta and update raccoon icon by @dgcampbe in https://github.com/Arcticons-Team/Arcticons/pull/2159
* Some Icons by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2162
* 50+ new icons (requested) by @wwwwwwari in https://github.com/Arcticons-Team/Arcticons/pull/2163
* Add 21 icons by @Npepperlinux in https://github.com/Arcticons-Team/Arcticons/pull/2164
* Added 13 Icon by @SagXD in https://github.com/Arcticons-Team/Arcticons/pull/2165
* Ukrainian-English Dictionary by @savetier in https://github.com/Arcticons-Team/Arcticons/pull/2168
* appfilter update and Clock intent issue fix by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2173
* Site update by @Donnnno in https://github.com/Arcticons-Team/Arcticons/pull/2182
* fix hamburger menu by @Donnnno in https://github.com/Arcticons-Team/Arcticons/pull/2183
* Added 19 new icons by @mrfenyx in https://github.com/Arcticons-Team/Arcticons/pull/2181
* Appfilter updates by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2179
* 100+ new icons and a few fixes by @wwwwwwari in https://github.com/Arcticons-Team/Arcticons/pull/2185
* Games Category by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2187
* Add 2 icons by @ayecptn in https://github.com/Arcticons-Team/Arcticons/pull/2188
* update three icons and add three icons by @dgcampbe in https://github.com/Arcticons-Team/Arcticons/pull/2184
* Added CUE by @Robro0-625 in https://github.com/Arcticons-Team/Arcticons/pull/2167
* Add icon for new Translink mLink app by @James-McK in https://github.com/Arcticons-Team/Arcticons/pull/2171
* Add LeipzigMOVE by @fwinn in https://github.com/Arcticons-Team/Arcticons/pull/2176
* some icons by @Donnnno in https://github.com/Arcticons-Team/Arcticons/pull/2192
* Update contribution guide. by @duese in https://github.com/Arcticons-Team/Arcticons/pull/2197
* Contributer image by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/pull/2190
* Remove dbna icon by @autinerd in https://github.com/Arcticons-Team/Arcticons/pull/2193
* Added 13 new icons, updated heyOBI by @mrfenyx in https://github.com/Arcticons-Team/Arcticons/pull/2194
* One new icon + 2 intent filters by @alchemiker in https://github.com/Arcticons-Team/Arcticons/pull/2199
* Add 6 icons and update appfilter by @Crazy-Unicooorn in https://github.com/Arcticons-Team/Arcticons/pull/2198
* 5 icons  by @skedastically in https://github.com/Arcticons-Team/Arcticons/pull/2200
* Add 4 icons and Update appfilter.xml by @Npepperlinux in https://github.com/Arcticons-Team/Arcticons/pull/2196
* new graphics by @Donnnno in https://github.com/Arcticons-Team/Arcticons/pull/2202
* New 25 icons by @wwwwwwari in https://github.com/Arcticons-Team/Arcticons/pull/2203

## New Contributors
* @mrfenyx made their first contribution in https://github.com/Arcticons-Team/Arcticons/pull/2181
* @Robro0-625 made their first contribution in https://github.com/Arcticons-Team/Arcticons/pull/2167
* @James-McK made their first contribution in https://github.com/Arcticons-Team/Arcticons/pull/2171
* @fwinn made their first contribution in https://github.com/Arcticons-Team/Arcticons/pull/2176

**Full Changelog**: https://github.com/Arcticons-Team/Arcticons/compare/9.7.4...10.0.5

<hr /><em>This discussion was created from the release <a href='https://github.com/Arcticons-Team/Arcticons/releases/tag/10.0.5'>Arcticons 10</a>.</em>

**Comment by Crazy-Unicooorn** [comment]
Thanks everyone!

@Donnnno there is a typo with the link to the Knowledge Base, an extra `https://` :
> https://https://docs.arcticons.com/

(both in release and discussion)

**Reply by Donnnno** [reply]
uh oh
